packages/components/Molecules/Form/fields/fieldRploptionbutton.vue

```diff
@@ -42,6 +42,7 @@ export default {
   $rpl-option-button-label-color: rpl-color('extra_dark_neutral') !default;
```

Comment:
Hi @dylankelly I think this `$rpl-form-element-text-color` can be used to set your form element text color https://github.com/dpc-sdp/ripple/blob/3358427e782379eae15f500b0a8aaa34a30169f1/packages/components/Molecules/Form/scss/_form.scss#L11

Reply:
✅